client/task-list/tasks/payments/components/WCPayCard/WCPayCard.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import {
+	Button,
+	Card,
+	CardBody,
+	CardHeader,
+	CardFooter,
+	__experimentalText as Text,
+} from '@wordpress/components';
+import interpolateComponents from 'interpolate-components';
+import { Link } from '@woocommerce/components';
+import { __ } from '@wordpress/i18n';
+import NoticeOutlineIcon from 'gridicons/dist/notice-outline';
+import { recordEvent } from '@woocommerce/tracks';
+
+/**
+ * Internal dependencies
+ */
+import './WCPayCard.scss';
+import WCPayAcceptedMethods from '../WCPayAcceptedMethods';
+import WCPayLogo from '../../images/wcpay-logo';
+
+export default ( { method, handleClick, isBusy } ) => {
```

Comment:
Maybe we could just pass `isEnabled` here?  Do we need the whole `method`?

Reply:
That _would_ have been the case, although we're now using a number of properties to pass to the `<Action />` component.